frontend/src/scenes/funnels/funnelLogic.ts

```diff
@@ -1317,6 +1344,28 @@ export const funnelLogic = kea<funnelLogicType<openPersonsModelProps>>({
             }
         },
     }),
+    urlToAction: ({ actions }) => ({
+        '/insights': (
+            _,
+            __,
+            {
+                funnel_advanced,
+            }: {
+                funnel_advanced?: boolean
+            }
+        ) => {
+            if (funnel_advanced) {
```

Comment:
If none of the advanced options are on (aka advancedOptionsUsedCount === 0) we should set advanced mode to false.